src/libs/CurrencyUtils.js

```diff
@@ -128,4 +128,25 @@ function convertToDisplayString(amountInCents, currency = CONST.CURRENCY.USD) {
     });
 }
 
-export {getCurrencyDecimals, getCurrencyUnit, getLocalizedCurrencySymbol, getCurrencySymbol, isCurrencySymbolLTR, convertToBackendAmount, convertToFrontendAmount, convertToDisplayString};
+/**
+ * Checks if passed currency code is a valid currency based on currency list
+ *
+ * @param {String} currencyCode
+ * @returns {Boolean}
+ */
+function isValidCurrencyCode(currencyCode) {
+    const currency = lodashGet(currencyList, currencyCode);
```

Comment:
Why lodashGet? Can we use another function as we just want to check if code exists in the list

Reply:
`currencyList` is an object with currency codes as fields, that is why I am using `lodashGet` which makes getting specific currency data much easier. 
<img width="566" alt="Screenshot 2023-09-20 at 14 57 25" src="https://github.com/Expensify/App/assets/59907218/3f612ca5-a04e-4bbf-a481-2e86a327ee15">

Reply:
But we can use `currencyList[currencyCode]` if we want to

Reply:
> I can't find currency code now in URL. Is it removed from URL?

By default currency code is not in url, only if you change the currency it will appear in the url.

> Also, could you please merge main
Merged

Reply:
Yes, we should return boolean. I see that. Cool, we're good to go with current implementation then